alembic.ini

```diff
@@ -28,9 +25,9 @@ sourceless = true
 
 # the output encoding used when revision files
 # are written from script.py.mako
-output_encoding = utf-8
+# output_encoding = utf-8
 
-sqlalchemy.url = postgres://postgres@localhost:8001/config
+sqlalchemy.url = postgres://postgres@localhost:5432/config
```

Comment:
Is the change to this file really intended or was that just a change that happened for your own setup?